mtx: [
       [508.069, 0.0, 332.821],
       [0.0, 506.445, 242.984],
       [0.0, 0.0, 1.0]
]


dist: [
    [
      -0.4146402,
      0.2407384,
      0.0, 0.0,
      -0.1040042
    ],
]

width_img: 640
height_img: 480
